server/views/components/customDisplay/crm14/crm14Income.njk

```diff
@@ -1,10 +1,18 @@
 {% from "../../formSubsection.njk" import formSubsection %}
+{% from "../../formField.njk" import formField %}
 
 {% macro crm14Income(data) %}
     <!-- Question 1 -->
     {% set fieldsQ1 = [
         { label: "Do you or your partner receive any of the benefits listed here?", value: data.receiveBenefits }
     ] %}
+
+    {% if data.proofBenefits === 'Yes' %}
+        {% set _ = fieldsQ1.push({ label: "Do you have any proof of receipt of benefits", value: data.proofBenefits + " (you will need to complete the rest of the application attaching evidence to demonstrate the receipt of your passported benefits)" + '<br> <br>' + "N.B. It is in your interests to provide evidence demonstrating that you receive one of the (passported) benefits listed above, if your application is processed as a non-passported assessment and your case progresses to the Crown Court you may be liable to pay a contribution towards the cost of your case from any Capital or Equity you may have"}) %}
+    {% else %}
+        {% set _ = fieldsQ1.push({ label: "Do you have any proof of receipt of benefits", value: data.proofBenefits + "No (enter details of your benefits income in the next section to enable your application to be processed as a non-passported assessment)"+ '<br> <br>' + "N.B. It is in your interests to provide evidence demonstrating that you receive one of the (passported) benefits listed above, if your application is processed as a non-passported assessment and your case progresses to the Crown Court you may be liable to pay a contribution towards the cost of your case from any Capital or Equity you may have"}) %}
```

Comment:
I don't think you need + "No .."  As data.proofBenefits will be equal to "No"

Reply:
`proofBenefits` is a String, We are getting `Yes` & Empty string `" "` values, 
I’m treating empty string as `No` and applied the logic.

Reply:
Yes understood that in UAT it is in empty string, are you sure that we are not getting proofBenefits = "No" in production? If we do, we will have "No No" in the label text.

Reply:
Thanks, just fixed and pushed it, plz check